### DISCUSSION TITLE
Artemis Operator and Broker upgrade

### DISCUSSION THREAD

**Discussion by lordigon** [arkmq-org/arkmq-org-broker-operator#842] [Q&A]
Hi all,
I installed a k8s cluster with Artemis Operator v 1.0.16 and installed an Arteis MQTT Broker via CRD using an Artemis Borker 1.0.23 with 2 nodes and persistence enabled.
I would like to upgrade the operator and the Artemis Broker to the latest version so I would like to know if there a specific procedure to do that to avoid breaking the actual configuration/installation.
Is there a specific strategy for this kind of upgrades? Any documentation to follow?

**Comment by brusdev** [comment]
@lordigon you need to update the operator deployment before upgrading the broker version. I'm not aware of any upgrade issues for your use case.
